products/confluence.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Confluence
```

Comment:
```suggestion
title: Atlassian Confluence
```

Not sure, but since we're adding multiple projects - keeping them all grouped would be nice. Plus, the plans are all tied in together from looking at the docs.

Reply:
Ok, but in this case should Jira Software be renamed Atlassian Jira Software (much less common IMHO).

Reply:
`Software` seems like an odd suffix. Can we just use "Atlassian Jira"?